.travis.yml

```diff
@@ -1,16 +1,20 @@
 language: objective-c
-osx_image: xcode7
-env:
-  matrix:
-    - 'DESTINATION="platform=iOS Simulator,name=iPhone 6,OS=8.4"'
-    - 'DESTINATION="platform=iOS Simulator,name=iPhone 6,OS=9.0"'
+matrix:
+  include:
+    - osx_image: xcode7.2
+      env: 'SIMULATOR="name=iPad Air,OS=9.2"'
+    - osx_image: xcode7
+      env: 'SIMULATOR="name=iPhone 6 Plus,OS=9.0"'
+    - osx_image: xcode6.4
+      env: 'SIMULATOR="name=iPhone 4s,OS=8.4"'
+    - env: 'SIMULATOR="name=iPad 2,OS=7.1"' # Default: Xcode 6.1
 
 before_install:
   - xcrun simctl list
   - brew update || brew update
 install:
-  - brew install coreutils
+  - if [ -z "$(brew ls --versions coreutils)" ] ; then brew install coreutils ; fi
```

Comment:
I was seeing a bunch of "ERROR: coreutils is already installed", so this seemed like a reasonable way to avoid that error.